tests/utils/PolygonUtilsTest.cpp

```diff
@@ -68,41 +67,46 @@ TEST_P(MoveInsideTest, MoveInside2)
-    MoveInsideParameters(Point(100, 50), 20, Point(80, 50)), //Starting on the border.
-    MoveInsideParameters(Point(80, 50), 20, Point(80, 50)), //Already inside.
-    MoveInsideParameters(Point(110, 50), 0, Point(100, 50)), //Not keeping any distance from the border.
-    MoveInsideParameters(Point(110, 50), 100000, Point(-99900, 50)) //A very far move.
-));
+INSTANTIATE_TEST_CASE_P(
+  MoveInsideInstantiation,
+  MoveInsideTest,
+  testing::Values(MoveInsideParameters(Point(110, 110), 28, Point(80, 80)), // Near a corner, moving inside.
+                  MoveInsideParameters(Point(50, 110), 20, Point(50, 80)), // Near an edge, moving inside.
+                  MoveInsideParameters(Point(110, 110), -28, Point(120, 120)), // Near a corner, moving outside.
+                  MoveInsideParameters(Point(50, 110), -20, Point(50, 120)), // Near an edge, moving outside.
+                  MoveInsideParameters(Point(110, 105), 28, Point(80, 80)), // Near a corner but not exactly diagonal.
+                  MoveInsideParameters(Point(100, 50), 20, Point(80, 50)), // Starting on the border.
+                  MoveInsideParameters(Point(80, 50), 20, Point(80, 50)), // Already inside.
+                  MoveInsideParameters(Point(110, 50), 0, Point(100, 50)), // Not keeping any distance from the border.
+                  MoveInsideParameters(Point(110, 50), 100000, Point(-99900, 50)) // A very far move.
+                  ));
 
 TEST_F(MoveInsideTest, cornerEdgeTest)
 {
     const Point close_to(110, 100);
-    const Point supposed1(80, 80); //Allow two possible values here, since the behaviour for this edge case is not specified.
+    const Point supposed1(80, 80); // Allow two possible values here, since the behaviour for this edge case is not specified.
     const Point supposed2(72, 100);
     constexpr coord_t distance = 28;
     const ClosestPolygonPoint cpp = PolygonUtils::findClosest(close_to, test_square);
     const Point result = PolygonUtils::moveInside(cpp, distance);
 
     constexpr coord_t maximum_error = 10;
     ASSERT_TRUE(vSize(result - supposed1) <= maximum_error || vSize(result - supposed2) <= maximum_error)
-        << close_to << " moved with " << distance << " micron inside to " << result << " rather than " << supposed1 << " or " << supposed2 << ".\n"
-        << "\tPS: dist to boundary computed = " << vSize(cpp.location - result) << "; vs supposed = " << vSize(cpp.location - supposed1) << " or " << vSize(cpp.location - supposed2) << ".\n"
-        << "\tclosest point = " << cpp.location << " at index " << cpp.point_idx << ".";
+      << close_to << " moved with " << distance << " micron inside to " << result << " rather than " << supposed1 << " or " << supposed2
+      << ".\n"
+      << "\tPS: dist to boundary computed = " << vSize(cpp.location - result) << "; vs supposed = " << vSize(cpp.location - supposed1)
+      << " or " << vSize(cpp.location - supposed2) << ".\n"
```

Comment:
This used to be formatted such that each line of code corresponds to a line in the output. Breaking it up this way breaks that.